app-ios/Sources/SearchFeature/SearchFilterSectionView.swift

```diff
@@ -0,0 +1,54 @@
+import appioscombined
+import ComposableArchitecture
+import Event
```

Comment:
I think this import isn't needed. Please remove this 🙏🏼

Reply:
ping. Please remove this 🙏🏼

Reply:
I think the `Event` module is necessary in this module for a [preview](https://github.com/DroidKaigi/conference-app-2022/pull/815/files#diff-a59a79346c30bbe8cab364a7637baa7f89ab879293a9908d6d8bbfdaf9c12cd2R60-R62).
Do you have any ideas how to preview this component without this module?